tee-worker/litentry/pallets/identity-management/src/lib.rs

```diff
@@ -304,10 +307,14 @@ pub mod pallet {
 			id_graph
 		}
 
-		pub fn all_id_graph_hash() -> Vec<(Identity, H256)> {
-			IDGraphLens::<T>::iter_keys()
-				.map(|k| (k.clone(), H256::from(blake2_256(&Self::get_id_graph(&k).encode()))))
-				.collect()
+		// get the IDGraph hash of the given `who`
+		pub fn id_graph_hash(who: &Identity) -> Option<H256> {
+			let id_graph = Self::id_graph(who);
+			if id_graph.is_empty() {
```

Comment:
if `who` is not a prime identity but a leaf of the id_graph, would the result be empty too?

Reply:
> if `who` is not a prime identity but a leaf of the id_graph, would the result be empty too?

Yes, will be `None`